.build/images/Quartz64/quartz64_defconfig

```diff
@@ -661,7 +673,7 @@ CONFIG_NFT_SOCKET=m
 CONFIG_NFT_SYNPROXY=m
 CONFIG_NFT_TPROXY=m
 CONFIG_NFT_TUNNEL=m
-CONFIG_NFT_XFRM=m
+CONFIG_NFT_XFRM=n
```

Comment:
Same here, this is set in RPi and Debian kernel. So disabling it explicitly, might beak something. If this module must not be loaded in your case, blacklist it via `/etc/modprobe.d`.